docs/compiler.md

```diff
@@ -25,6 +25,8 @@ a dialect of Go rather than a complete port of the language:
     in variables and returning the result.
  * lambdas are supported, but closures are not.
  * maps are supported, but valid map keys are booleans, integers and strings with length <= 64
+ * converting value to interface type is a no-op, original value will be used,
```

Comment:
But why it is no-op? I thought that current commit code makes it possible.

Reply:
Maybe it's not phrased correctly, but in Go proper an interface value is created and then passed around, we don't do that in our compiler.

Reply:
I think we need to mention that interface conversion never panics and always "succeeds".

Reply:
Fixed.